src/app/components/showcase.tsx

```diff
@@ -1,6 +1,37 @@
+import { useEffect, useState } from 'react';
+import calculateScrollAlpha from '@/scripts/calculateScrollAlpha';
+
 export default function Showcase() {
+	const [alpha, setAlpha] = useState('0.00');
+	const [hidden, setHidden] = useState(true);
+
+	function handleScroll() {
+		let scrollAlpha: string = calculateScrollAlpha();
+
+		if (scrollAlpha === '0.00') {
+			setHidden(true);
+		} else {
+			setHidden(false);
+		}
```

Comment:
Could this be re-written as just `setHidden(scrollAlpha === '0.00');`?

Reply:
Yup - this is why I'm thankful for your code reviews haha. In the moment I couldn't think of writing this a better way, and I haven't used your suggested syntax enough (will now). 

But that makes perfect sense now to just set state to whatever the condition `scrollAlpha === '0.00'` outputs to (`true` or `false`). 

Cleaned up in a recent commit!